mmv1/third_party/terraform/transport/config.go.erb

```diff
@@ -1161,14 +1163,23 @@ func (c *Config) GetCredentials(clientScopes []string, initialCredentialsOnly bo
 			return *creds, nil
 		}
 
-		creds, err := transport.Creds(c.Context, option.WithCredentialsJSON([]byte(contents)), option.WithScopes(clientScopes...))
-		if err != nil {
-			return googleoauth.Credentials{}, fmt.Errorf("unable to parse credentials from '%s': %s", contents, err)
+		if c.UniverseDomain != "" && c.UniverseDomain != "googleapis.com" {
```

Comment:
Can we add tests where we explicitly set GDU? We could consider adding them beside the non-GDU tests.

Reply:
Good idea, will do.

Reply:
Thanks again for the suggestion, was able to catch some small bugs during this process.